src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CompilationContext.cs

```diff
@@ -440,13 +455,28 @@ private static string GetNextMethodName(ArrayBuilder<MethodSymbol> builder)
                 generateMethodBody);
         }
 
+        private EEMethodSymbol GetPseudoVariableMethod(
+            TypeNameDecoder<PEModuleSymbol, TypeSymbol> typeNameDecoder,
+            EENamedTypeSymbol container,
+            string methodName,
+            Alias alias)
+        {
+            var syntax = SyntaxFactory.IdentifierName("");
```

Comment:
Consider:  SyntaxFactory.IdentifierName(SyntaxFactory.MissingToken(SyntaxKind.IdentifierToken))

(or something similar that might indicate that the syntax is synthesized)